### DISCUSSION TITLE
雑談ルーム

### DISCUSSION THREAD

**Discussion by ymuichiro** [ymuichiro/symbol_japan_forum#16] [General]
なんか取り止めもない話がしたいとき

**Comment by nononon00** [comment]
いっちばんのり！🤩✨

**Reply by ymuichiro** [reply]
一番乗りいただいてもらいました！

**Comment by nononon00** [comment]
そういえばで書くんだけど、重要度を再計算するimportanceblockは720ブロック(約6時間)毎に訪れますが、そのタイミングで残高を戻す形で取引所と行き来させたらハーベストできるimportanceは計算されたタイミングの残高でおおよそ決まるのかな、って
本日Twitterでのやり取りでそんな話がでてました🤔
頻繁にトレードをしたいから取引所に置きたいけどハーベストもしたい場合、貼り付いてXYM行き来させれば可能なのかな？

**Reply by mikunNEM** [reply]
CrptyoBelieverさんに聞いてみたら、テストネットで試してみましょうって🥳
(180ブロック毎なのでテストしやすい)

10,000 XYM下回ると、ハーベストはすぐストップするっぽいので、10,000XYMはキープしつつですかね🤔

**Comment by ymuichiro** [comment]
Symbolコミュニティからコア開発者にお菓子やレッドブル用チップを送ろう企画の件（matsunokiさんTweet）、ちょっと面白めにしたく追加で企画を考えてみました。

期待するところとしては例えば この仕組みをコミュニティで作ってしまって（SSS_Extention等の組み合わせでいけるのでは）グローバルのレッドブルやお菓子メーカーに企画提案するのはどうかと思っています（窓口はTwitter or お問合せ窓口より）

勿論仕組みややり方はもっと詰める必要があるので、もし賛同してくれる方がいらっしゃいましたら表明 & アイディア出し手伝ってくれると嬉しいです

大きく出るなら世界のBlockChainによるギフト発行システムの前例が作れないかなと

https://docs.google.com/presentation/d/1H_w1LhMrOdVLYJrDDMkLq8vcDfWdzV0vqdJuanOHu44/edit?usp=sharing

**Reply by kurikou02** [reply]
チップを気軽に送れる仕組みの件、Bluetoothボタン買ってきてワンポチで1XYM送金するプログラムを書いてたんですが、SymbolのPython-SDKに苦戦して止まってます…。Bluetoothボタンの検知までは簡単なので、コミュニティのつよつよエンジニアの皆さんが書いたら一瞬だと思います。

事例として面白いのでIoTボタンでやってますが、将来的にはフォーラムサイトかWikiかclusterか、コミュニティが集まるWeb空間上にボタン置いて、SSSで各自が送れるようになるのがベストですよね。更に、現金ではなくモノの方が面白いと思うので、どっかのウォレットにプールされていって、一定額溜まったら自動でレッドブルが発注されてコアデブメンバに届く…とかでしょうか。

**Reply by ymuichiro** [reply]
とりあえずPythonの方は解決してよかったです
他の優先度高いやつの対応を進めていてこれはまだ進められていませんが...

3ヶ月スパンでQuestを立てて、3ヶ月単位で払い出ししたらいいんじゃないかと思ったりはしたり。
難点はどうやって菓子を買って届けるか、なんですよね

Symbolの特性上、Tokenの送信は簡単で、プールも既に出来る。
これをいかにしてレッドブル等のチケットに変えてしまうか？

この点はしっかりした企画を作ってレッドブルに提案しにいくか？とは思っています

**Reply by kurikou02** [reply]
シームレスに自動発注→モノ（レッドブル）で届く…までいけたらカッチョ良いですが、最初は誰かが取りまとめて換金→購入→配送でしょうか…。でも何かダサいかな…換金で税金リスクもありますもんね…。

**Reply by ymuichiro** [reply]
> でも何かダサいかな…換金で税金リスクもありますもんね…。

これ、その通りなんですよ
誰かがプールすると負担が集中する、税金もある、なので、普通にやろうとするとうまくいきそうにないんです

じゃぁどうするか？って私的にはLINE GIFTみたいな流れが現実的なのかなと思ったりしています
https://gift.line.me/about

**Reply by ymuichiro** [reply]
ただそれでも問題があって、受け取る側の個人的な連絡先や住所を公開する必要性が出てくれば参加者が生まれないので、ブロックチェーン上で表現できる仕組みが必要になってくると思っています。

これに向けてまずは私はそういった機能もあるウォレットをまずは作ろうかなと思っています
（最近Twitterで投稿したやつです。DIDの他にバーコード表示機能を作れないかなと
